pages/inspector.tsx

```diff
@@ -105,6 +107,10 @@ const Home: NextPage = () => {
       );
 
       setIsLoading(true);
+
+      const fetchedContractVersion = await getVersion(address, web3);
+      setContractVersion(fetchedContractVersion);
```

Comment:
this might need some try/catch logic - i think it might fail silently otherwise.